packages/http-client-java/generator/http-client-generator-test/src/main/java/com/_specs_/azure/core/page/implementation/PageClientImpl.java

```diff
@@ -814,8 +814,6 @@ public PagedIterable<BinaryData> listWithCustomPageModel(RequestOptions requestO
     }
 
     /**
-     * List with Azure.Core.Page&lt;&gt;.
-     * 
      * Get the next page of items.
      * <p><strong>Response Body Schema</strong></p>
```

Comment:
It seems all the changes is on "next" op (that created by us)? E.g. here "listWithPageNextSinglePageAsync".

We are fine to change (or even no doc) as these are internal.
But not sure why this only happen to "next" op (this seems not relates to template in tsp)?

Reply:
Thanks for the catch, yeah, this is another case where previous `description` and `detail` have difference from current `doc` and `summary`. But not related with template.
Previously `List with Azure.Core.Page&lt;&gt;.` exists on `detail`, but now it exists on `summary`. So from code-model But
previous:
<img width="686" alt="image" src="https://github.com/user-attachments/assets/eedf19df-d1c0-4cf7-85ac-ab172dafc12c">
now:
<img width="639" alt="image" src="https://github.com/user-attachments/assets/0ddd15e5-aa32-4193-be9c-882fa58f67fa">

It is  strange that it only impacts next operation, guess it's due to codegen logic, but I have to debug to find the cause.